TicTacToe.js

```diff
@@ -0,0 +1,320 @@
+//Readline bloiler plate code
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    readlineInterface.question(questionText, resolve);
+  });
+}
+
+let playerStatus = 1;
+rowOne = [' 1','|','2','|','3'];
+rowDashes = ['---','---','---'];
+rowTwo = [' 4','|','5','|','6'];
+rowThree = [' 7','|','8','|','9'];
+
+
+printBoard();
+console.log('\n' +  "Welcome to tic tac toe, a two player game.  The first player to guess will be X's and the second will be O's.  Please choose a game board section based on it corresponding number."+'\n')
+playerOne();
+
+//printBoard();
+
+function printBoard(){
+  let rowOneString = rowOne.join(' ');
+  console.log(rowOneString);
+  let rowOfDashes = rowDashes.join(' ');
+  console.log(rowOfDashes);
+  let rowTwoString = rowTwo.join(' ');
+  console.log(rowTwoString);
+  console.log(rowOfDashes);
+  let rowThreeString = rowThree.join(' ');
+  console.log(rowThreeString);
+}
+
+function testMove(testNumber,playerNumber){
+  testNumber = String(testNumber).trim();
+  if(testNumber == 1){
```

Comment:
This function functions -- which is good! -- but it's got a lot of duplication in it. Now that you have it working, try to make it more concise and readable by using loops or variables or a sub-function. Think about what is common between all of these "else if" blocks, and extract the differences into variables. 

This step I'm suggesting is called "refactoring" and it's Step Two of the old programmer's maxim: 

1. Make it work
2. Make it right
3. Make it fast ... 

In that order. And don't do step 3 :-) .